modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/ui/pages/main/DLCContentPage.java

```diff
@@ -54,6 +55,36 @@ public String deleteFunction() throws IOException {
     }
 
     /**
+     * Delete all the messages in DeadLetter Channel
+     */
+    public void deleteAllDLCMessages() {
+
+        String deletingMessageID = null;
+
+        try {
+            deletingMessageID = driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                     .getElement("mb.dlc.first.message.id"))).getText();
+            if(StringUtils.isNotBlank(deletingMessageID)) {
+                log.info("delete all dlc messages");
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.table.choose.all.box.xpath")))
+                                                    .click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.table.delete.button"))).click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.function.confirm"))).click();
+                driver.findElement(By.xpath(UIElementMapper.getInstance()
+                                                    .getElement("mb.dlc.browse.function.success"))).click();
+            }
+
+        }catch (Exception e) {
+            log.error("unable to find any valid messages in dead letter channel.");
```

Comment:
IMO adding a log in catch will make it more descriptive and readable than without a log. AFAIK there's no other way except for using try catch in this situation(please refer these[1][2]). 
What i did was find first message in the dlc table and if that element exist proceed with bulk purge messages from dlc. If there's no element present in first row of the table it will throw NoSuchElementException which will catch and log the error as there's no elements to delete.

[1] http://stackoverflow.com/questions/20503713/check-if-a-table-exists-or-not
[2] http://stackoverflow.com/questions/20536850/org-openqa-selenium-nosuchelementexception-unable-to-locate-element/